src/app/models/attribute-translation.ts

```diff
@@ -0,0 +1,15 @@
+import { Resource } from './resource';
+
+export class AttributeTranslation {
+  id?: number;
+  key: string;
+  'onesky-phrase-id': string;
+  required: boolean;
+  resource?: Resource;
+}
+
+export interface IPromises {
```

Comment:
this interface name could probably use something more descriptive, it could conflict with some other functionality in the future that uses a similar promise mechanism to do batch updates.

Maybe `IAttributeTranslationPromises`

Reply:
I've renamed it `IAttributeTranslationPromises`